docs/advanced/components.md

```diff
@@ -57,4 +57,6 @@ Decidim.register_component(:my_component) do |component|
 end
 ```
 
-Every model in a component doesn't have to (and should not) know about its parent participatory process, but instead should be scoped to the components.
+Every model in a component doesn't have to (and should not) know about its parent participatory space, but instead should be scoped to the components.
+
+## Gotchas with engines
```

Comment:
This section seems to be missing!

Reply:
Ups! I moved it into the modules.md file. Fixing it!